test/extensions.txt

```diff
@@ -150,7 +150,7 @@ Hello!
 
 | _abc_ | セン |
 | ----- | ---- |
-| this row has a space at the end | | 
+| this row has a space at the end | |
```

Comment:
```suggestion
| this row has a space at the end | | 
```

This was probably removed by accident.

Reply:
This is "whitespace trimming" on vscode :heart: